cypress/integration/selectors.ts

```diff
@@ -2,15 +2,16 @@ type Cy = Cypress.Chainable;
 
 const selectors = {
   search: {
-    findSearchInput: (cy: Cy) => cy.get("#datasetSearch"),
-    findIncludeDraftsCheckbox: (cy: Cy) => cy.get("#dataset-include-drafts"),
+    searchInput: (cy: Cy) => cy.get("#datasetSearch"),
+    draftsCheckbox: (cy: Cy) => cy.get("#dataset-include-drafts"),
+  },
+  panels: {
+    left: (cy: Cy) => cy.get('[data-name="panel-left"]'),
+    right: (cy: Cy) => cy.get('[data-name="panel-right"]'),
   },
   edition: {
-    findLeftPanel: (cy: Cy) => cy.get('[data-name="panel-left"]'),
-    findRightPanel: (cy: Cy) => cy.get('[data-name="panel-right"]'),
-    findConfiguratorFilters: (cy: Cy) =>
-      cy.findByTestId("configurator-filters"),
-    findChartFiltersList: (cy: Cy) => cy.findByTestId("chart-filters-list"),
+    configFilters: (cy: Cy) => cy.findByTestId("configurator-filters"),
+    chartFilters: (cy: Cy) => cy.findByTestId("chart-filters-list"),
   },
 };
```

Comment:
🙇

Reply:
I think also an option to consider would be to not have callbacks here (e.g. `selectors.search.searchInput: "dataset-search"`). It would require us to convert all of them to use test-ids, which would be good thing I believe, and then we could do `cy.findByTestId(selectors.search.input)`, for example. But this is not a scope of this PR and I am not sure if this would be better, let me know what you think :)

Reply:
I think for some selectors we can use findByLabelText or findByPlaceholder, this is why I chose to support the full `cy` object here. IMHO this is better that test id since we do not have to add them, and test ids should only be an escape hatch when we have no proper user seeable thing that can be targeted.

Reply:
Right, I didn't think about selecting by text (and I based the comment on [this section](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements), where they recommend to use custom data attributes). But I'm fine with the current approach, just wanted to hear your opinion, thanks!

Reply:
I am basing myself on the guiding principles of testing library, not of Cypress. This is why I added the methods from testing library. I think it's better if we do not have to add a lot of data attributes everywhere, and test as much with "human" visible attributes.

https://testing-library.com/docs/queries/about#priority